packages/warn/src/index.ts

```diff
@@ -7,11 +7,11 @@
  * For example: stats warns when any tracks events aren't properly formatted (@see lib/analytics)
  */
 
-let warn;
+let warn: typeof console.warn;
```

Comment:
The type of `console.warn` is the very generic `( ...data: any[] ) => void`, and for clarity, I would probably write it verbatim here.